doc/lsst.ap.pipe/pipeline-tutorial-gen2.rst

```diff
@@ -0,0 +1,265 @@
+.. py:currentmodule:: lsst.ap.pipe
+
+.. _pipeline-tutorial-gen2:
+
+###############################
+Running the AP pipeline (Gen 2)
+###############################
+
+Setup
+=====
+
+Pick up where you left off in :doc:`Getting Started <getting-started-gen2>`.
+This means you already have a repository of ingested DECam data and have setup
+the LSST Science Pipelines stack as well as ``ap_pipe`` and ``ap_association``.
+
+Your directory structure should look something like
+
+.. code-block:: none
+
+   repo/   <-- where you will be running ap_pipe
+      registry.sqlite3   <--- created during image ingestion
+      image1.fits   <--- linked here by default during image ingestion
```

Comment:
```suggestion
      image1.fits   <--- linked here (or to a date subdirectory) during image ingestion
```

Reply:
Is the "date subdirectory" for a specific instrument? I'm worried about being too detailed here.

Reply:
All of this is already overly specific for DECam so it doesn't matter much either way 🙃